dlt/destinations/dataset/ibis_relation.py

```diff
@@ -173,6 +194,94 @@ def _get_filtered_columns_schema(self, columns: Sequence[str]) -> TTableSchemaCo
             # here we just break the column schema inheritance chain
             return None
 
+    def _filter_using_root_table(
+        self,
+        key: str,
+        values_to_include: Union[list[Any], list["ReadableIbisRelation"], "ReadableIbisRelation"],
+    ) -> "ReadableIbisRelation":
+        """Filter the root table and propagate to current table."""
+        # remove the proxy to facilitate transformations
+        if isinstance(values_to_include, ReadableIbisRelation):
+            values_to_include = values_to_include._ibis_object
+
+        # Case 1: if current table is root table, you can filter directly and exit early
+        dlt_root_table = get_root_table(self.schema.tables, self.table_name)
+        if dlt_root_table["name"] == self.table_name:
+            # doing ibis.expr.types.Table[COL_NAME] returns a column, but ReadableIbisRelation[COL_NAME] returns a table
```

Comment:
I think you can access columns with self.columns(column_name) but I am not sure. In any case it would be nice to have a simple helper method for accessing columns so it is clearer what is going on.